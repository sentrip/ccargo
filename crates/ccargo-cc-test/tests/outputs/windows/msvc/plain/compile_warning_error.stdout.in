main.c
main.c(4,1): error C2220: the following warning is treated as an error
main.c(4,1): warning C4133: 'initializing': incompatible types - from '__int64 *' to 'int *'
main.c(5,1): warning C4133: 'initializing': incompatible types - from '__int64 *' to 'int *'
