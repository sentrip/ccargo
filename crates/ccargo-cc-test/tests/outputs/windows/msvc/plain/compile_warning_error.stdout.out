error C2220: the following warning is treated as an error
   --> main.c:4:1
error C4133: 'initializing': incompatible types - from '__int64 *' to 'int *'
   --> main.c:4:1
error C4133: 'initializing': incompatible types - from '__int64 *' to 'int *'
   --> main.c:5:1
